agenthub/codeact_agent/prompt.py

```diff
@@ -0,0 +1,222 @@
+from opendevin.sandbox.plugins import SWEAgentCommandsRequirement
+
+_SWEAGENT_BASH_DOCS = '\n'.join(
+    filter(
+        lambda x: not x.startswith('submit'),
+        SWEAgentCommandsRequirement.documentation.split('\n')
+    )
+)
+# _SWEAGENT_BASH_DOCS content below:
```

Comment:
Please take this as a grain of salt: my understanding is this paragraph is auto-generated, and you copy & paste here for developers' understanding. There are two reasons again doing this:

1) The content could be found in integration test artifacts.
2) It's hard to keep this comment in sync with its real source.

That being said, this is handy for people to understand, so I don't have a strong opinion.

Reply:
Good point! Our project is still under active development; so i guess for now this may benefits devs more in terms of clarity. Once we finalize everything, we can figure out a better way to display this / remove this?

Reply:
Sounds fair